codeHF/dpl-config_run3.json

```diff
@@ -153,23 +153,12 @@
         "processRun3": "false"
     },
     "tof-event-time": {
-        "param-file": "",
```

Comment:
This needs to be removed to stop crashing according to Nicolo: https://mattermost.web.cern.ch/alice/pl/srdmx9wupfgkdr1qy3rieu9pze

Reply:
Should be fixed in the most recent version.